packages/time-picker/src/time-picker-com/panel-time-range.vue

```diff
@@ -1,76 +1,69 @@
 <template>
-  <transition
-    name="el-zoom-in-top"
-  >
-    <div
-      v-if="visible"
-      class="el-time-range-picker el-picker-panel"
-    >
-      <div class="el-time-range-picker__content">
-        <div class="el-time-range-picker__cell">
-          <div class="el-time-range-picker__header">{{ t('el.datepicker.startTime') }}</div>
-          <div
-            :class="{ 'has-seconds': showSeconds, 'is-arrow': arrowControl }"
-            class="el-time-range-picker__body el-time-panel__content"
-          >
-            <time-spinner
-              ref="minSpinner"
-              role="start"
-              :show-seconds="showSeconds"
-              :am-pm-mode="amPmMode"
-              :arrow-control="arrowControl"
-              :spinner-date="minDate"
-              :disabled-hours="disabledHours_"
-              :disabled-minutes="disabledMinutes_"
-              :disabled-seconds="disabledSeconds_"
-              @change="handleMinChange"
-              @set-option="onSetOption"
-              @select-range="setMinSelectionRange"
-            />
-          </div>
-        </div>
-        <div class="el-time-range-picker__cell">
-          <div class="el-time-range-picker__header">{{ t('el.datepicker.endTime') }}</div>
-          <div
-            :class="{ 'has-seconds': showSeconds, 'is-arrow': arrowControl }"
-            class="el-time-range-picker__body el-time-panel__content"
-          >
-            <time-spinner
-              ref="maxSpinner"
-              role="end"
-              :show-seconds="showSeconds"
-              :am-pm-mode="amPmMode"
-              :arrow-control="arrowControl"
-              :spinner-date="maxDate"
-              :disabled-hours="disabledHours_"
-              :disabled-minutes="disabledMinutes_"
-              :disabled-seconds="disabledSeconds_"
-              @change="handleMaxChange"
-              @set-option="onSetOption"
-              @select-range="setMaxSelectionRange"
-            />
-          </div>
+  <div v-if="actualVisible" class="el-time-range-picker el-picker-panel">
```

Comment:
Can you take a look? @zazzaz I am not sure whether it is a good solution. I try to remove  `v-if` for better performance, but some errors will be thrown, such as `can not read xx from undefined`. I suppose there are something wrong with the default value, but the component are toooo complicated. Could you help me out？